source/element-kinds.md

```diff
@@ -314,18 +314,22 @@ Element parameter groups associated with this element kind are:
 
 Zero length element to mark a particular position.
 The main purpose of this element is to name a position in the beamline.
+`Marker` elements has a unit transport map. That is, a particle's phase space coordinates
+are not altered when with passage through the element
 
 Element parameter groups associated with this element kind are:
 - [**ApertureP**](#s:aperture.params): Aperture parameters.
 - [**BodyShiftP**](#s:bodyshift.params): Orientation of element with respect to its nominal position.
 - [**FloorP**](#s:floor.params): Floor position and orientation.
 - [**MetaP**](#s:meta.params): Meta parameters.
 - [**ReferenceP**](#s:ref.params): Reference parameters.
-- [**ReferenceChangeP**](#s:ref.change.params): Reference energy change and/or reference time correction.
```

Comment:
Can you comment why we want to remove `ReferenceChangeP` but not other attributes, e.g., `FloorP` or `TrackingP`?

Reply:
`FloorP` parameters are generally computed quantities. This is something that should be made explicit in the documentation (See #107).

`ApertureP` can be taken out and then `Aperture` elements can be used. What do people think?